docker/Dockerfile-opflex-build

```diff
@@ -1,5 +1,5 @@
-FROM noiro/opflex-build-base
-ARG BUILDOPTS="--with-static-boost --with-boost=/usr/local/boost_1_58_0"
+FROM challa/opflex-build-base:demo
+ARG BUILDOPTS=
```

Comment:
Can we go back to using noiro/opflex-build-base with the static linking (that is how we are shipping today)?

Reply:
Reverted